lua/autorun/lua_reloadent.lua

```diff
@@ -64,9 +64,9 @@
 --   1.0   - First public release                            --
 ---------------------------------------------------------------
 
-AddCSLuaFile("lua_reloadent.lua")
-local FindInLua = VERSION < 150 and file.FindInLua or function(path)
-	return file.Find(path, LUA_PATH)
+AddCSLuaFile()
+local FindInLua = function(path)
+    return file.Find(path, "LUA")

```

Comment:
You changed tabs to spaces there, can you fix that and ``--amend`` your commend?